### DISCUSSION TITLE
Failed to resolve callback.

### DISCUSSION THREAD

**Discussion by retry-the-user** [deepgram/community#512] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

This is the JSON I'm getting back TO MY CALLBACK URL:

{"err_code":"Bad Request","err_msg":"Failed to resolve callback.","request_id":"9022de36-f4f2-429f-bdd5-bba1bda37c27"}

How are you failiing to resolve the callback if you're sending a response TO my callback endpoint?  Ridiculous.  Stop making YOUR errors into my errors.

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

9022de36-f4f2-429f-bdd5-bba1bda37c27

### If possible, please attach an example audio file to reproduce the issue.

The issue appears to be intermittent, since plenty of other STT requests were successful today.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by retry-the-user** [comment]
I got three of these errors in 3 minutes. Here are the other two JSON responses :

{"err_code":"Bad Request","err_msg":"Failed to resolve callback.","request_id":"49ec3ac5-9b98-469d-a54a-c9d122c186a9"}
{"err_code":"Bad Request","err_msg":"Failed to resolve callback.","request_id":"a83de3c5-6e14-4816-8e5f-d8049a6cc73a"}

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @retry-the-user, our apologies for the errors. We had a five-minute window in which a DNS issue led us to improperly process callbacks. Agreed that the messaging doesn't make sense when we're sending the callback error to your callback URL. Please retry these requests as needed; it can be helpful to have an exponential backoff retry strategy.
